packages/docs/modules/page-config/blocks/change-log/index.ts

```diff
@@ -0,0 +1,14 @@
+import { definePageConfigBlock } from '../../types'
```

Comment:
Would be possible to fix in context of this pr?

<img width="939" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/0d634b1b-ebf1-4212-b091-77fd7dfcf042">

Reply:
Must be fixed already.